### PR TITLE
Issue 191: add template for ISSUE and PR

### PR DESCRIPTION
**Pull request by jiazhai**
Following BP-9, provide a template for ISSUE and PR.
When create issue, will use ISSUE template; while create PR will use PR template.

### CODE REVIEW

**Comment by sijie** [comment]
@jiazhai I think these two files introduced the rate-check warnings. please either exclude them from apache-rat checking or add apache license header.

**Comment by jiazhai** [comment]
@sijie Done the fix, excluded .github dir.
Thanks a lot.

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+Is this a question, feature request, or bug report?
+
+**QUESTION**
+
+Have you checked our documentation at http://bookkeeper.apache.org/ , If you could not find an answer there, please consider asking your question in our community forum at dev@bookkeeper.apache.org, as it would benefit other members of our community.
```
Comment (eolivelli) [inline .github/ISSUE_TEMPLATE.md]: Maybe it is better to cite usr@ and not @dev

**Comment by eolivelli** [comment]
+1 I will try  the new script at #194  againts this pr

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (jiazhai) [inline .github/ISSUE_TEMPLATE.md]: sure, will change it to usr@. Thanks.

**Comment by eolivelli** [comment]
@sijie for you is ok ? so that we can merge and start using it for new issues/pr ?

**Comment by sijie** [comment]
LGTM +1

**Comment by sijie** [comment]
merging it now.

**Pull request closed without merging**